examples/Celement_with_env_1.hs

```diff
@@ -3,16 +3,19 @@ module Concept where
 import Tuura.Concept.STG
 
 -- C-element with environment circuit described using signal-level concepts
-circuit a b c = interface <> outputRise <> inputFall <> outputFall <> inputRise <> initialState
+
+data Sign = A | B | C deriving (Eq, Enum, Bounded)
+
+circuit = interface <> outputRise <> inputFall <> outputFall <> inputRise <> initialState
```

Comment:
Should this be named `system` instead?

Reply:
Yes it also should, reminding me that I haven't set anything to check for `system`